server/src/main/resources/log4j.properties

```diff
@@ -20,6 +20,11 @@ log4j.rootLogger=DEBUG, A1
 # A1 is set to be a ConsoleAppender.
 log4j.appender.A1=org.apache.log4j.ConsoleAppender
 
+log4j.logger.org.apache.zookeeper=DEBUG
```

Comment:
Are these log level changes leftover from debugging?

Reply:
This is only for Zookeeper module.